lms/envs/production.py

```diff
@@ -694,6 +694,7 @@ def get_env_setting(setting):
 GOOGLE_ANALYTICS_TRACKING_ID = AUTH_TOKENS.get('GOOGLE_ANALYTICS_TRACKING_ID')
 GOOGLE_ANALYTICS_LINKEDIN = AUTH_TOKENS.get('GOOGLE_ANALYTICS_LINKEDIN')
 GOOGLE_SITE_VERIFICATION_ID = ENV_TOKENS.get('GOOGLE_SITE_VERIFICATION_ID')
+GOOGLE_ANALYTICS_4_ID = AUTH_TOKENS.get('GOOGLE_ANALYTICS_4_ID')
```

Comment:
I don't see why it could hurt, but I'm pretty sure the majority of our tracking will come out from the MFE rather than the LMS level. Do you know what kind of events are tracked and sent from the LMS? Ben and Mike should be able to determine if they will actually be valuable or just noisy.